appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -434,12 +443,37 @@
                          registeredComponentName eventName)
                        #f))))
 
+       ;; For the HandlesEventDispatching interface
+       (define (dispatchMultiEvent eventName :: java.lang.String
```

Comment:
Suppose you have 3 buttons. With the third block,

![screen shot 2015-04-12 at 1 11 37 pm](https://cloud.githubusercontent.com/assets/4552296/7106618/26158a12-e115-11e4-95dc-b2f46d1dc741.png)

when you click any of the 3 buttons, the text of that button will change to "Clicked"

With the second block, a block like this

![screen shot 2015-04-12 at 1 13 12 pm](https://cloud.githubusercontent.com/assets/4552296/7106622/63b0c404-e115-11e4-932a-ac6b40e1ceea.png)

will only handle events for Button1. If Button1 is clicked, it's text changes.